unity-renderer/Assets/UnityGLTF/Scripts/Editor/GLTFImporter.cs

```diff
@@ -533,6 +533,7 @@ private GameObject CreateGLTFScene(string projectFilePath)
                 loader.addMaterialsToPersistentCaching = false;
                 loader.initialVisibility = true;
                 loader.useMaterialTransition = false;
+                loader.maxTextureSize = 512;
```

Comment:
im wary about this hardcoded limit, can you reference to `DataStore.i.textureSize.gltfMaxSize`?

Reply:
This is the GLTFImporter for the Editor. We should never reference DataStore here, it won't be initialized anyway =(

Reply:
ok then!